tests/unit/views/HelloWorld.test.js

```diff
@@ -54,9 +57,9 @@ describe('HelloWorld.vue', () => {
 
     test('Render default contents', async () => {
       const wrapper = await setupWrapper();
+      const normalizedHtml = normalizeAmp(wrapper.html());
```

Comment:
```suggestion
```

Reply:
Done, switched back to short version:
       `expect(wrapper.html()).toMatchSnapshot();`